src/main/java/codeu/controller/BotServlet.java

```diff
@@ -129,11 +129,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
         messageStore.addMessage(message);
 
         try {
+            // TODO Check why here is nullpointer in the test
 			actionMatcher.matchAction(cleanedMessageContent, username, response);
 		} catch (PersistentDataStoreException e) {
```

Comment:
Instead of creating a separate catch you can do

`} catch  (PersistantDataStoreException | NullPointerException e)  {`